style_guide.md

```diff
@@ -0,0 +1,76 @@
+# Jsonnet Style Guide
+This style guide documents how we want to handle jsonnet files when building the schemas with consistency. For more general principles and styling rules go to [Databricks repository](https://github.com/databricks/jsonnet-style-guide#databricks-jsonnet-guide).
+
+## Folder structure
+
+### England/Wales and Northern Ireland
+
+- Jsonnet source files are separated into two folders: `england-wales` and `northern-ireland` with a number of sub-folders, each for corresponding schemas (e.g. `ccs`, `communal-establishment`, `household`, `individual`). These are divided into blocks in `blocks` folder.
+
+### Top level Jsonnet files in these folders
+
+- For each schema there is a lists of block and placeholder imports, as well as schema structure divided into sections, named after given schema and ending with `.jsonnet` suffix.
+
```

Comment:
I think some of this content would be helped with a visualisation of the directory structure, something like:
```
england-wales/
   ccs/
      blocks/
      lib/
         rules.libsonnet
   communal-establishment/
   household/
   individual/
   lib/
      common_rules.libsonnet
      placeholders.libsonnet
   ccs_household.jsonnet
   census_communal_establishment.jsonnet
   census_household.jsonnet
   census_individual.jsonnet
ireland/

```

Reply:
I added it now, thanks!👍